metadata.yaml

```diff
@@ -28,9 +28,17 @@ notes: |-
   https://scholar.google.com/citations?hl=en&user=inVqu_4AAAAJ
```

Comment:
The `notes` should be updated. See https://github.com/ACCESS-NRI/access-om2-configs/issues/64

We've not made this a QA check, but we could (should?).